webapp/views/Guest/Guest.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 
 import ModuleSwitch from '@webapp/components/moduleSwitch'
-import ResetPasswordView from '@webapp/guest/resetPassword/resetPasswordView'
-import ForgotPasswordView from '@webapp/guest/forgotPassword/forgotPasswordView'
+import ResetPasswordView from '@webapp/views/Guest/views/ResetPassword/ResetPassword'
+import ForgotPassword from '@webapp/views/Guest/views/ForgotPassword/ForgotPassword'
```

Comment:
add index.js so that imports become
```
import ResetPasswordView from '@webapp/views/Guest/views/ResetPassword'
 import ForgotPasswordView from '@webapp/guest/forgotPassword'
```